<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickSettingsWidget</class>
 <widget class="QWidget" name="QuickSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="colorModelBox">
     <property name="title">
      <string>Color Model</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="btnHSV">
        <property name="text">
         <string>HS&amp;V</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btnHSL">
        <property name="text">
         <string>HSL</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btnHSI">
        <property name="text">
         <string>HSI</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btnHSY">
        <property name="text">
         <string>HSY'</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
